### PR TITLE
User/getById

### PR DESCRIPTION
**Pull request by festinalente91**
- added user get by id

### CODE REVIEW

**Comment by HewonJeong** [comment]
lgtm

File: src/server/auth/models.js [hunk 1]
```diff
@@ -51,20 +46,19 @@ const AuthModel = {
...
-          facebookId: id
-        });
-        return UserModel.getUserKey(idxKey);
-      })
-      .then(userKey => {
-        return {
-          userKey,
-          userType: USER_TYPE.FACEBOOK
-        };
+      .then(facebookId => {
```
Comment (ovekyc) [inline src/server/auth/models.js:51]: Why don't use . chaining?

**Comment by ovekyc** [comment]
LGTM

File: src/server/auth/models.js (hunk 1 above)
Comment (festinalente91) [reply src/server/auth/models.js:51]: like how?

**Comment by hewonjeong** [comment]
lgtm

**Pull request merged**